tests/test_config_flow.py

```diff
@@ -57,8 +58,9 @@ async def test_init_entry(hass):
             CONF_HOST: "hostname",
             CONF_LOCAL_KEY: "localkey",
             CONF_POLL_ONLY: False,
-            CONF_PROTOCOL_VERSION: "auto",

```

Comment:
Do all these changes from "auto" to 3.3 mean that the auto protocol version detection is no longer working after this change?

Reply:
According to the tinytuya documentation, there is no auto protocol version detection.
If no protocol version is passed when creating the device, tinytuya initializes it to 3.1
`########################################################
#             Core Classes and Functions
########################################################

class XenonDevice(object):
    def __init__(
            self, dev_id, address=None, local_key="", dev_type="default", connection_timeout=5, version=3.1, persist=False, cid=None, node_id=None, parent=None # pylint: disable=W0621
    ):`